src/query/ast/src/parser/statement.rs

```diff
@@ -1322,6 +1327,35 @@ pub fn unset_source(i: Input) -> IResult<UnSetSource> {
     )(i)
 }
 
+pub fn set_var_hints(i: Input) -> IResult<HintItem> {
+    map(
+        rule! {
+            SET_VAR ~ ^"(" ~ ^#ident ~ ^"=" ~ #subexpr(0) ~ ^")"
+        },
+        |(_, _, name, _, expr, _)| HintItem { name, expr },
+    )(i)
+}
+
+pub fn any_token(i: Input) -> IResult<&[Token]> {
+    Ok((i.slice(1..), i.0.slice(1..)))
+}
```

Comment:
it's better to move it to `src/query/ast/src/util.rs`
```suggestion
pub fn any_token(i: Input) -> IResult<&Token> {
    move |i| match i.0.get(0).filter(|token| token.kind != TokenKind::EOI) {
        Some(token) => Ok((i.slice(1..), token)),
        _ => Err(nom::Err::Error(Error::from_error_kind(
            i,
            ErrorKind::Other("expected any token but reached the end"),
        ))),
    }
}
```